<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2119</width>
    <height>1459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Pixelmap" name="ui_label_video">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="ui_textedit_console">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1090</y>
      <width>951</width>
      <height>341</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Pixelmap" name="ui_label_flow">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>0</y>
      <width>1024</width>
      <height>1024</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>1080</y>
      <width>721</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="ui_button_calc">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_vpath">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>/home/user/Videos/image.gif</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_vselect">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_device">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Device:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ui_combo_device">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_vload">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_vunload">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Unload</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_ppath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Save path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_ppath">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>/home/user/test_pokes.json</string>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_vpath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Video:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_psave">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_pload">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_pselect">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>210</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_pclear">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_button_pclear_all">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ui_checkbox_rel_flow">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Relative Flow</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_edit_chunk_size">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ui_label_chunk_size">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>170</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Chunk Size</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="ui_slider_t">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1050</y>
      <width>1021</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui_edit_t">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>1040</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_button_set_t">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>1040</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_button_next_t">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>1040</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QSlider" name="ui_slider_shiftx">
    <property name="geometry">
     <rect>
      <x>2060</x>
      <y>0</y>
      <width>20</width>
      <height>1021</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="ui_slider_shifty">
    <property name="geometry">
     <rect>
      <x>2080</x>
      <y>0</y>
      <width>20</width>
      <height>1021</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="ui_slider_zoom">
    <property name="geometry">
     <rect>
      <x>1849</x>
      <y>1050</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TickPosition::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>1050</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui_label_pos">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>1040</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>[PosX, PosY]</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Pixelmap</class>
   <extends>QLabel</extends>
   <header>src/widgets/pixel_map</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
